web/src/hooks/UseOrganization.ts

```diff
@@ -23,3 +30,122 @@ export function useOrganization(name: string) {
     organization,
   };
 }
+
+export function useNamespaceAutoPrunePolicies(
```

Comment:
How about adding a new hook for all AutoPrune related functions? Might be better to separate it out.